agent/unit-test/configstore_unittest.cpp

```diff
@@ -96,12 +85,12 @@ TEST_F(ConfigStoreTest, constructorDir) {
 /* Constructing a FilteredConfigStore with a directory path,
  * even if all files can not be parsed, should not throw an exception.
  */
-/* This test fails intermittently, not clear why
-TEST_F(FilteredConfigStoreTest, constructorDir2) {
+//This test fails intermittently, not clear why
+TEST_F(ConfigStoreTest, constructorDir2) {
     // No config files, but ok dir
     ASSERT_NO_THROW(FilteredConfigStore("/home/vagrant/softwarecontainer/"));
```

Comment:
I agree. We shouldn't have an absolute path here. As the description suggests the purpose of the test was to use a directory we had the rights to read from and which contains many files, preferably no json files.